examples/llms/providers/customProvider.ts

```diff
@@ -77,6 +80,10 @@ export class CustomLLM extends LLM<CustomLLMOutput, CustomGenerateOptions> {
     return { tokenLimit: Infinity };
   }
 
+  async embed(input: LLMInput[], options?: EmbeddingOptions): Promise<EmbeddingOutput> {
```

Comment:
`// eslint-disable-next-line unused-imports/no-unused-vars`  not included here because it results in an `Unused eslint-disable directive` warning

<img width="849" alt="image" src="https://github.com/user-attachments/assets/cbb995b5-c78a-4c00-ae11-3dc2aa0a337e">